appveyor.yml

```diff
@@ -0,0 +1,17 @@
+version: 1.0.{build}
+init:
+- git config --global core.autocrlf true
+install:
```

Comment:
This section would be required for building the solution directly, but since we're running the build script (**build.ps1**) [it's unnecessary](https://github.com/openstacknetsdk/openstack.net/blob/90e9228b30f6b0f787cb8d67832511437ed272cc/build/build.ps1#L40).